src/scriptfuncs.cpp

```diff
@@ -10238,6 +10238,25 @@ static DROID_TEMPLATE *scrCheckTemplateExists(SDWORD player, DROID_TEMPLATE *psT
 	return nullptr;
 }
 
+// deprecated
+bool scrWeaponShortHitUpgrade()
+{
+	SDWORD					player, weapIndex;
```

Comment:
The file doc/CodingStyle.md discourages using types like SDWORD.

Reply:
Taking inspiration from the rest of the nearby code. This is a useless wzscript function like the one below it which I added only for consistency reasons.